routes/v1.js

```diff
@@ -81,7 +81,7 @@ module.exports = function(appObj) {
 
     return {
         path: '/',
-        api_version: 1,
```

Comment:
domain i understand, but why remove api_version?  Are we not versioning the API?